src/index.html

```diff
@@ -6,8 +6,32 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="container">
+    <div data-qa="card" class="card">
+      <img src="images/imac.jpeg" alt="iMac"
+      class="card__image"
+      >
```

Comment:
Need to check and fix it everywhere
<img width="1037" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/b9d859e0-5690-4a33-9e17-7cfe7f19eb05">